src/java.base/share/classes/java/lang/VirtualThread.java

```diff
@@ -106,6 +105,21 @@ final class VirtualThread extends BaseVirtualThread {
      *  TIMED_PARKED -> UNPARKED        // unparked, may be scheduled to continue
      *  TIMED_PINNED -> RUNNING         // unparked, continue execution on same carrier
      *
+     *   RUNNING -> BLOCKING       // blocking on monitor enter
+     *  BLOCKING -> BLOCKED        // blocked on monitor enter
+     *   BLOCKED -> UNBLOCKED      // unblocked, may be scheduled to continue
```

Comment:
Does this mean it now owns the monitor, or just it is able to re-contest for monitor entry?

Reply:
It means it is scheduled to run again and re-contest for the monitor.